src/univariate/continuous/ksdist.jl

```diff
@@ -73,12 +73,12 @@ function cdf_durbin(d::KSDist,x::Float64)
     k, ch, h = ceil_rems_mult(n,x)
 
     m = 2*k-1
-    H = Array(Float64,m,m)    
+    H = Array(Float64,m,m)
     for i = 1:m, j = 1:m
-        H[i,j] = i-j+1 >= 0 ? 1.0 : 0.0        
+        H[i,j] = i-j+1 >= 0 ? 1 : 0
     end
-    r = 1.0
-    for i = 1:m 
+    r = 1
```

Comment:
Since `h` is a the same type as `x` this will get promoted to a `Float64` in the loop so it should be initialized differently. Should this function be parametric?